packages/base-driver/lib/protocol/routes.js

```diff
@@ -303,6 +303,14 @@ const METHOD_MAP = /** @type {const} */ ({
       payloadParams: {required: ['orientation']},
     },
   },
+  // w3c v2 https://www.w3.org/TR/webdriver2/#get-computed-role
+  'session/:sessionId/element/:elementId/computedrole': {
+    GET: {},
```

Comment:
do we not want to go ahead and add command names in case a driver wants to implement these? otherwise there's not much point in having them here?

Reply:
what about current changes?